### DISCUSSION TITLE
v8.0.0

### DISCUSSION THREAD

**Discussion by MurhafSousli** [MurhafSousli/ngx-scrollbar#404] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/MurhafSousli/ngx-scrollbar/releases/tag/v8.0.0'>v8.0.0</a>.</em>

**Comment by f4irline** [comment]
Has anyone else ran into problems with Jest and ngx-scrollbar 8.0.0? I'm getting the following errors related to ngx-scrollbar after updating to 8.0.0:

![image](https://user-images.githubusercontent.com/31544493/132814271-2e1374f9-7101-41c9-a881-f9ff77ccc734.png)

With ngx-scrollbar 7.6.1 everything seems to be fine. It's _possibly_ a configuration issue, but I also tried just adding ngx-scrollbar to a clean project with clean Jest installation using [@briebug/jest-schematic](https://www.npmjs.com/package/@briebug/jest-schematic), and have the same issue with the clean project as well.
